### PR TITLE
Provide details about block editor label syntax

### PR DESCRIPTION
**Pull request by glombek**
As suggested by HQ staffers after I published this blog post https://joe.gl/ombek/blog/umbraco-angularjs-filter-cheat-sheet/

### CODE REVIEW

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+---
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md:2]: Which major versions of Umbraco does this article apply to?

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# Configuring block editor label properties
+
+When configuring a block, the label property allows you to define a label for the appearance of the Block in the editor. The label can use AngularJS template string syntax to display values of properties. Example: `My Block {{myPropertyAlias}}` will be shown as: `My Block FooBar`.
+
+You can also use more advanced expression using AngularJS filters, e.g. `{{myPropertyAlias | limitTo:100}}` or for a property using Richtext editor `{{myPropertyAlias | ncRichText | truncate:true:100}}`.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md:8]:
```suggestion
You can also use more advanced expressions using AngularJS filters. Example: `{{myPropertyAlias | limitTo:100}}` or for a property using Richtext editor `{{myPropertyAlias | ncRichText | truncate:true:100}}`.
```

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+It is also possible to use properties from the settings model by using `{{$settings.propertyAlias}}`.
+
+## Useful Angular filters
+
+As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/filter), Umbraco ships with some additional filters which are useful for setting the label field of block editors.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md:14]:
```suggestion
As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/filter), Umbraco ships with some additional filters which are useful for setting the Label field of Block editors.
```

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+| [umbWordLimit](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbWordLimit) | Truncates to a number of words (rather than characters) | String                      | n: maximum number of words in string                         |
+| [umbCmsTitleCase](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsTitleCase) | Converts a string to title case                         | String                      |                                                              |
+| [umbCmsJoinArray](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsJoinArray) | Joins an array into one string                          | Array (of string or object) | separator: string used to join values together, e.g. "`, `"<br />prop (optional): string key indicating which property to join when used on an array of objects |
+
+### Custom filters
+
+If the above filters don't suit your needs, you can create custom filters by creating a plugin in `App_Plugins` and adding a filter module.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md:28]:
```suggestion
If the filters do not suit your needs, you can create custom filters by creating a plugin in `App_Plugins` and adding a filter module.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @glombek ! 💪 

I think it makes sense to have a separate article about these filters 💪 

I've made a few suggestions to grammar etc.
Will send it by our developers for a check-up as well!

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md (hunk 1 above)
Comment (glombek) [reply Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md]: NC was brought in in 7.7, so I believe all apply from then

**Review by nielslyngsoe — APPROVED** [review]
@sofietoft Looks good to me :-)

**Comment by sofietoft** [comment]
Thanks for the review @nielslyngsoe ! 😄 

Looks like we've got the green lights to merge this one in @glombek ! 💪 
Thanks again for adding this info to our docs - much appreciated!

**Pull request merged**